script/build.js

```diff
@@ -1,7 +1,16 @@
 import {join} from 'path'
-import {terser} from "rollup-plugin-terser"
+import {terser} from 'rollup-plugin-terser'
 import size from 'rollup-plugin-size'
 
+const emitPackageScopeFile = (type) => {
```

Comment:
this is required so that commonjs/umd file can be treated as commonjs script while having `.js` extension (otherwise it gets treated as a module due to the closest package.json - the root one - having `"type":"module"`)